### DISCUSSION TITLE
Can we omit defaultpassword when generating debug txt?

### DISCUSSION THREAD

**Discussion by benyaminl** [sumatrapdfreader/sumatrapdf#2415] [Q&A] [answered]
Hello, I just see my old debug.txt, and delete it from internet, I seen my own private password is there, is there any plain in future the DefaultPassword value won't be dumped on? So We don't need to skimp on it each time we need to submit the dump? Thanks.

**Comment by kjk** [comment, marked as answer]
It's not impossible but too much effort to implement.

**Reply by benyaminl** [reply]
Uhm is that really hard to implement? I wanna try

**Reply by kjk** [reply]
I assume you're talking about crash reports that contain a copy of preferences.
The easiest way to implement this would probably be to post-process the generated text of preferences and do sort of string replace of password with spaces.

I'll be happy to merge a change if you make a Pull Request.
